docs/FeatureManagement/Samples/NoFilters.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "BooleanTrueFeatureFlag",
+                "description": "A simple feature flag, that always returns true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "BooleanFalseFeatureFlag",
+                "description": "A simple feature flag, that always returns false.",
+                "enabled": false,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringTrueFeatureFlag",
+                "description": "A simple feature flag, that always returns false.",
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringFalseFeatureFlag",
+                "description": "A simple feature flag, that always returns true.",
+                "enabled": "false",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "InvalidEnabledFeatureFlag",
+                "description": "A simple feature flag, that has an invalid enabled field.",
+                "enabled": "invalid",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "MinimalFeatureFlag",
+                "enabled": true
+            },
+            {
+                "id": "NoEnabled"
```

Comment:
```suggestion
                "id": "NoEnabledFeatureFlag"
```

Reply:
I think this is an open question in general. Do we or do we not include the word Feature Flag in the names? Can't it be assumed that they are all feature flags? I'm good with either way, but I was slightly leaning towards no including `FeatureFlag` in every feature flag name.

Reply:
I am ok with not including `FeatureFlag` in every feature flag name. But we should make the name style consistent.